adserver/forms.py

```diff
@@ -658,6 +703,9 @@ def __init__(self, *args, **kwargs):
         self.fields["end_date"].help_text = _(
             "The target end date for this flight (it may go after this date)"
         )
+        self.fields["auto_renew"].help_text = _(
+            "Flights that automatically renew receive an additional 10% discount"
```

Comment:
I do wonder about updating our published prices to assume auto-renewal, perhaps even with a toggle like many pricing pages have between monthly/annual pricing.

Reply:
I would make it a toggle. I think when this rolls out we also update the marketing pages.